Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14.15-buster as builder
```

Comment:
This Docker file is a pain.  It seems that the version of docker used by circleci build the docker image doesn't work very well with multistage builds. In theory this build can use node:14.16-buster-slim as 'base'. do `RUN apt-get update && apt-get upgrade -y` and then use this image for building the app (FROM base as builder) and also for creating the final release image (FROM base as release). That would be faster and simpler. Unfortunately this breaks the build at the mkdir /app step.  Clearly the builder and release images aren't separate.
Hence the version below. meh.